docs/topics/gradle/gradle-compilation-and-caches.md

```diff
@@ -31,14 +40,14 @@ There are several ways to disable incremental compilation:
 
   The parameter should be added to each subsequent build.
 
-Note: Any build with incremental compilation disabled invalidates incremental caches. The first build is never incremental.
+Disabling incremental compilation invalidates incremental caches. The first build is never incremental.
```

Comment:
The previous version makes sense to me, because the incremental cache becomes invalid only after the build. Do you think it might be too clunky?

Reply:
I think it's a bit too clunky - especially starting with Note: feels weird as normally the rest of the docs uses admonition in this case. 

We could try to phrase it more explicitly, how about:
"When you disable incremental compilation, incremental caches become invalid after the build." 

alternatively, perhaps:
"When incremental compilation is disabled, incremental caches are invalidated after the build."